README.md

```diff
@@ -34,42 +34,44 @@ container_vulns = lw.vulnerabilities.get_container_vulnerabilities(image_digest=
 ## How-To
 
 The following data points are required to instantiate a LaceworkClient instance:
-   - `account`: The Lacework account/organization domain. (`xxxxx`.lacework.net)
-   - `subaccount`: (Optional) The Lacework sub-account domain. (`xxxxx`.lacework.net)
-      - This is only used if leveraging the Manage@Scale organization feature of Lacework
-   - `api_key`: The API Key that was generated from the Lacework UI/API.
-   - `api_secret`: The API Secret that was generated from the Lacework UI/API.
+
+- `account`: The Lacework account/organization domain. (`xxxxx`.lacework.net)
+- `subaccount`: (Optional) The Lacework sub-account domain. (`xxxxx`.lacework.net)
+  - This is only used if leveraging the Manage@Scale organization feature of Lacework
+- `api_key`: The API Key that was generated from the Lacework UI/API.
+- `api_secret`: The API Secret that was generated from the Lacework UI/API.
 
 To generate an API Key and Secret, do the following:
-   1. In the Lacework web interface, go to Settings -> API Keys
-   2. Create a new API Key, or download information for an existing one.
+
+1.  In the Lacework web interface, go to Settings -> API Keys
+2.  Create a new API Key, or download information for an existing one.
 
 ### Environment Variables
 
 The `account`, `subaccount`, `api_key`, and `api_secret` can also be set using environment variables.
 
-| Environment Variable | Description | Required |
-|----------------------|-------------|:--------:|
-|`LW_ACCOUNT`|Lacework account/organization domain (i.e. `xxxxx`.lacework.net)|Y|
-|`LW_SUBACCOUNT`|Lacework sub-account domain (i.e. `xxxxx`.lacework.net)|N|
-|`LW_API_KEY`|Lacework API Access Key|Y|
-|`LW_API_SECRET`|Lacework API Access Secret|Y|
+| Environment Variable | Description                                                      | Required |
+| -------------------- | ---------------------------------------------------------------- | :------: |
+| `LW_ACCOUNT`         | Lacework account/organization domain (i.e. `xxxxx`.lacework.net) |    Y     |
+| `LW_SUBACCOUNT`      | Lacework sub-account domain (i.e. `xxxxx`.lacework.net)          |    N     |
```

Comment:
Thanks for following the standard from the go-sdk. 

![tenor-163685544](https://user-images.githubusercontent.com/5712253/108582397-4dec9580-72f0-11eb-88d1-8059cd194b61.gif)